lib/resources/r4/financial_transaction_create.yaml

```diff
@@ -0,0 +1,325 @@
+
+- name: code
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  description: Identifies the 'type' of resource - equivalent to the resource name for other resources.
+  action: create
+  example: |
+    {
+      "code": "fintrans"
+    }
+  note: This will always be fintrans
+  url: http://hl7.org/fhir/r4/basic-definitions.html#Basic.code
+
+- name: financial-transaction-type
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: Category of transaction Payment Type Codes (Example), Whether this is a payment or adjustment.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-type",
+      "valueCode": "Adjustment"
+    }
+  note: Will only accept payment or adjustment
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-type
+
+- name: financial-transaction-transaction-alias
+  required: 'Yes'
+  cardinality: 1..1
+  type: BackboneElement
+  description: Backbone element of the transaction aliases.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-transaction-alias",
+      "extension": [
+          {
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-transaction-alias-alias",
+              "valueString": "729813"
+          }
+      ]
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-transaction-alias
+
+- name: financial-transaction-transaction-alias-alias
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The aliases used to create the FinancialTransaction resources.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-transaction-alias",
+      "extension": [
+          {
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-transaction-alias-alias",
+              "valueString": "729813"
+          }
+      ]
+    }
+  note: A value that correlates to a client defined name which represents the combination of the type, subtype, and reason describing the FinancialTransaction. Is required to create al FinancialTransaction resource
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-transaction-alias-alias
+
+- name: financial-transaction-location
+  required: 'No'
+  cardinality: 0..1
+  type: string
+  description: The location where the payment was collected.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-location",
+      "valueString": "201802160003"
+    }
+  note: These are client configured location values.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-location
+
+- name: financial-transaction-method
+  required: 'C'
+  cardinality: 0.1
```

Comment:
Update and add missing dot

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/a1a39411fbb35b2ccf79e521ffae5e6ae92d62b9#diff-26870e1b94fc9955e130f3f940dbe2e7R98

Reply:
+1